configs/Wisun_Stm_s2lp_RF_lab.json

```diff
@@ -44,6 +44,7 @@
         "uc-fixed-channel": "0xffff",
         "bc-fixed-channel": "0xffff",
         "network-name": "\"ARM-WS-LAB-NWK\"",
+        "GTK0": null,        
```

Comment:
Do we need these defaults be present here?

Reply:
Removed nulls from .json files, added GTK0 setting to readme.